### ISSUE TITLE
Minimum Target Size, custom elements and user agent exception

### ISSUE THREAD

**Issue by gundulaniemann** [w3c/wcag#3998]
In the user agent exception example it should be made clear that it always applies as long as the size still is default.

The exception in the normative text says:
"User agent control: The size of the target is determined by the user agent and is not modified by the author;"

So it talks about not manipulating the size. It does not talk about any other attribute a control might have, like color.

Yet the example still might be misunderstood as "if any attribute is altered, the exception does not apply".
Therefore is should say:
"User agent control: Browsers have default renderings of some controls, such as the days of the month calendar in an <input type="date">. As long as the author has not modified the user agent default size, the target size for a User agent control is excepted."

(The word 'size' was added after 'user agent default'.)

Some background for scrollbars:
The default size for scrollbars usually is 'auto'. This way any setting a browser or an extension might offer can be applied. This remains unchanged if for example the colors of the scrollbars have been changed by the author.
Only the size needs to remain untouched.

**Comment by yatil** [comment]
No, it is meant that once you take ownership of the element by applying any styles to it, you are also responsible to meet the minimum sizing requirements. You can only use this exception if you leave all styling to the browser. This is consistent with similar exceptions in other success criteria.

**Comment by scottaohara** [comment]
But I think @gundulaniemann makes a very good point, @yatil.

per the quoted text ("User agent control: The size of the target is determined by the user agent and is not modified by the author;"), I don't think it's clear at all that changing any style would undo the exception if the original sizing wasn't changed.  The text as written differently than other instances where the exception is more overtly nullified if any styling changes are made.

for example, the following two SCs are much clearer that the original UA appearance/visual presentation has an exception, but if the appearance is changed, then it becomes an author responsibility.

1.4.11's language for user interface components states:
>Visual information required to identify [user interface components](https://www.w3.org/TR/WCAG22/#dfn-user-interface-components) and [states](https://www.w3.org/TR/WCAG22/#dfn-states), except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author;

1.4.13 has this exception:
>Exception: The visual presentation of the additional content is controlled by the user agent and is not modified by the author.


2.5.5 and 2.5.7's exceptions are the same, and "size" is specifically called out, not any and all visual appearance as other SCs mention.   Whether one agrees with whether that should be the case or not, the fact the text is different and is scoped specifically to "size", I don't think there's a strong case anyone could make based on the normative text that any style change beyond those that would impact the size would undo the exception.

**Comment by patrickhlauke** [comment]
agree with @scottaohara and @gundulaniemann here too, the language seems more specifically about size not being modified

**Comment by yatil** [comment]
@patrickhlauke @scottaohara @gundulaniemann My bad, you’re right. That said, I think the wording in the SC is sufficient to convey this.

**Comment by gundulaniemann** [comment]
Glad that you agree.
nevertheless I have experienced several misunderstanding, therefore I still suggest to add the word 'size' into the example to make clear that manipulating the size (or nit) is the crucial point.
